app/web/features/communities/events/LongEventCard.tsx

```diff
@@ -1,145 +1,164 @@
-    },
-    padding: theme.spacing(1),
-    [theme.breakpoints.up("sm")]: {
-      padding: theme.spacing(2),
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    margin: 0,
+    "&:not(:first-child)": {
+      margin: theme.spacing(2, 0),
     },
+    border: `1px solid ${theme.palette.grey[300]}`,
+    borderRadius: theme.spacing(1),
   },
-  overviewContent: {
-    display: "grid",
-    gap: theme.spacing(1),
-    gridTemplateAreas: `
-      "titles image"
-      "content content"
-    `,
-    gridTemplateColumns: "1fr 1fr",
-    [theme.breakpoints.up("md")]: {
-      gridTemplateColumns: "2fr 1fr",
+  attendees: {
+    display: "flex",
+    alignItems: "flex-end",
+    justifyContent: "flex-end",
+    minWidth: theme.spacing(10),
+  },
+  card: {
+    display: "flex",
+    width: "100%",
+    height: theme.spacing(20),
+    [theme.breakpoints.down("xs")]: {
+      height: "auto",
     },
   },
-  eventInfoContainer: {
-    gridArea: "titles",
+  cardMedia: {
+    height: "100%",
+    width: "25%",
+    objectFit: "fill",
   },
-  eventTimeContainer: {
-    alignItems: "center",
+  cardContent: {
+    width: "75%",
     display: "flex",
-    marginBlockStart: theme.spacing(1),
+    flexDirection: "column",
+    justifyContent: "space-between",
   },
-  attendeesCountContainer: {
-    alignItems: "center",
+  eventInfo: {
     display: "flex",
+    justifyContent: "flex-end",
+    flexDirection: "column",
   },
-  icon: {
-    display: "block",
-    fontSize: "1.25rem",
-    lineHeight: 1.5,
-    marginInlineEnd: theme.spacing(0.5),
+  row: {
+    display: "flex",
+    justifyContent: "space-between",
   },
-  image: {
-    objectFit: ({ eventImageSrc }) =>
-      eventImageSrc === eventImagePlaceholderUrl ? "contain" : "cover",
-    height: 80,
-    [theme.breakpoints.up("md")]: {
-      height: 120,
+  tags: {
+    minWidth: theme.spacing(15),
+    [theme.breakpoints.down("xs")]: {
+      minWidth: theme.spacing(10),
     },
-    width: "100%",
-    gridArea: "image",
   },
-  onlineOrOfflineInfo: {
-    fontWeight: "bold",
-    color: theme.palette.grey[600],
-  },
-  content: {
-    gridArea: "content",
+  title: {
+    display: "-webkit-box",
+    lineClamp: 2,
+    boxOrient: "vertical",
+    overflow: "hidden",
+    textOverflow: "ellipsis",
+    maxHeight: "3em" /* Approximate height for 2 lines of text */,
+    lineHeight: "1.5em",
+    paddingRight: theme.spacing(2),
   },
 }));
 
-export interface LongEventCardProps {
+const LongEventCard = ({
+  event,
+  userId,
+}: {
   event: Event.AsObject;
-}
-
-export default function LongEventCard({ event }: LongEventCardProps) {
-  const { t } = useTranslation([COMMUNITIES]);
+  userId: number | null | undefined;
+}) => {
   const classes = useStyles({
     eventImageSrc: event.photoUrl || eventImagePlaceholderUrl,
   });
-  const theme = useTheme();
-  const isBelowLg = useMediaQuery(theme.breakpoints.down("md"));
+  const { t } = useTranslation([COMMUNITIES]);
```

Comment:
shouldn't we use [EVENTS] key here?

Reply:
There is no `EVENTS` namespace. It's within the `COMMUNITIES` namespace. You can see it in `namespaces.js`.